### ISSUE TITLE
ERROR " Account has insufficient balance for requested action."

### ISSUE THREAD

**Issue by MatDal** [yasinkuyu/binance-trader#258]
Hello,
after placing the sell order the script enters a loop with an error "Account has insufficient balance for requested action." even if the order has not been closed.
How can I solve?
Thank you

**Comment by GitHubEmploy** [comment]
This usually happens when your account does not have enough balance, have you looked at how much you are buying, and if you have the sufficient funds?

**Comment by aavesh1** [comment]
@GitHubEmploy  i have sufficient balance but it's still stuck at

Started...
Trading Symbol: DOGEBTC
Buy Quantity: 4347.00000000
Stop-Loss Amount: 0
Profit Mode Options:
        Preferred Profit: 0.00%
        Buy Price : (Bid+ --increasing 0.00000001)
        Sell Price: (Ask- --decreasing 0.00000001)

**Comment by GitHubEmploy** [comment]
Yes, I had the same problem, are you using paper trading or real money?
